x-pack/plugins/security_solution/public/common/components/landing_cards/index.tsx

```diff
@@ -94,7 +94,6 @@ export const LandingCards = memo(() => {
           <EuiFlexItem>
             <iframe
               allowFullScreen
-              allowTransparency
```

Comment:
This is for the video component on the getting started page right? Is this a fix? I'm not seeing a difference with/without this attribute on chrome. 🤷‍♂️

Reply:
Hrmmm, was thinking it was theme related, but seeing the white bg with or without this attribute, so not sure, haha 😅 

<img width="1707" alt="image" src="https://user-images.githubusercontent.com/2946766/216450759-da907dc5-566a-4a40-b30a-5b0e9ea99aaa.png">

Reply:
@stephmilovic, you should know. I'm not sure about this change.